BoardView.cpp

```diff
@@ -28,6 +28,11 @@ BoardView::~BoardView() {
 	free(m_lastFileOpenName);
 }
 
+void BoardView::ShowError(char* msg) {
+	m_lastErrorMsg = msg;
```

Comment:
Great, this refrac should make it easier to drop `ImGui::OpenPopup` for errors and just use native dialogs for all errors.

Would mean wrapping `MessageBox`, and Gtk + Cocoa equivalents in a cross-platform `show_error` function though, so happy to leave it for now.